de.metas.adempiere.adempiere/base/src/main/java-legacy/org/adempiere/model/RelationTypeZoomProvider.java

```diff
@@ -108,10 +124,15 @@ public String toString()
 		final ZoomProviderDestination source = sourceAndTarget.getLeft();
 		final ZoomProviderDestination target = sourceAndTarget.getRight();
 
-		if (!source.matchesAsSource(zoomSource))
+		final boolean isReferenceTarget = target.isReferenceTarget();
+
+		if (!isReferenceTarget)
```

Comment:
please reduce the nesting by combining those ifs

Reply:
Obsolete after refactoring